gen/src/Gen/Types/Service.hs

```diff
@@ -53,14 +44,15 @@ data Signature
   deriving (Eq, Show, Generic)
 
 sigToText :: Signature -> Text
-sigToText V2 = "v2"
-sigToText _ = "v4"
+sigToText = \case
+  V2 -> "v2"
+  _ -> "v4"
```

Comment:
Is this correct? Should `V3`, `V3HTTPS` and `S3` all return `"v4"`?

Reply:
It's probably not correct but I'll worry it about it separately.